.travis.yml

```diff
@@ -14,10 +14,6 @@ cache: false
 addons:
   chrome: stable
   firefox: latest
-  sauce_connect:
-    username: OktaSignInWidget
-  jwt:
-    secure: Yacq1Un4yBSsFNpoZz3ut2PJ8Ac46hSuBt1RZczuMzDpJwTt8jUhe8XvHDhMAQk1kZ3R7x3B1xwaDqGWAN62OzsC4uvT6b6EibCDq9cWMNUtXu7RWvf7oQg4rEZ0cl6PYmI9dxg4CYD0yc1G7+WVG16eIiisuPKHpQ7GoXEGCQV8i5Nf8DWu64fRcIVs+JSbDbC49IQdyPLoUmUyLESLhDLmsY2YVzbAjtm35X1p5SZrsqEK3AP4Iz64zuzBnuuKIrq1+N+yD+jZGyGwZU2BzCG6Zb7DE2jSLpLdXhLZtQPhU0Q6dYMQAoenINPwcMD3ZeWp7oo4LK/CCS7FTrL0f0CfQ75luid8f+SAde4i4Rg1OizKiAgClj1GkZKesNJBY8K1MUlWjSHHsvgY9ZwnXufemYYNq4YA/NwXFjkgFDbH+49YwfV2k2Imk7oMCYqRQaidyZvqLn5jf8+anINMk53qkDj95FT/0lcrc+fmSvhf+v3xXjLT6fK4rSIyxSS8PO0RBNQUb/h7Jv6Y21+esvhPwIjsxRR8j/YNgWPkqGffh4AYBgeObIUdOYWZaGQiYfRz4XA6FKZzANBjuSwSFWu6pgZvBHVM1/tx6zPZ98rmo9T6ipdgj3kHhzvU5PbJJ/ndUztRmEF1Kc6a8nyvmZlx1UnXoQBnBjLFySAfYjU=
```

Comment:
`SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` variables have been moved to travis settings